src/index.html

```diff
@@ -1,13 +1,336 @@
+                  src="./images/photos/detail-1.png"
+                  alt="auto-unlock"
+                  class="details__image"
+                >
+
+              <img
+                  src="./images/photos/detail-2.png"
+                  alt="auto-unlock"
+                  class="details__image"
+                >
+            </div>
+
+            <h3 class="details__title">Auto Unlock </h3>
+
+            <p class="details__text">
+              The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+            </p>
+          </article>
+          <article class="details__part">
+            <div class="details__images-container">
+              <img
+                  src="./images/photos/detail-3.png"
+                  alt="Range-lights"
+                  class="details__image"
+                >
+
+              <img
+                  src="./images/photos/detail-4.png"
+                  alt="Integrated-lights "
+                  class="details__image"
+                >
+            </div>
+
+            <h3 class="details__title">Range & Integrated lights </h3>
+
+            <p class="details__text">
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </article>
+          <article class="details__part">
+            <div class="details__images-container details__images-container--left-image--small">
+              <img
+                  src="./images/photos/detail-5.png"
+                  alt="Hydraulic-disc-brakes"
+                  class="details__image"
+                >
+              <img
+                  src="./images/photos/detail-6.png"
+                  alt="Lightweight"
+                  class="details__image"
+                >
+            </div>
+
+            <h3 class="details__title">Hydraulic disc brakes & Lightweight </h3>
+
+            <p class="details__text">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </article>
+          <a href="#" class="button">Explore</a>
+        </div>
+      </section>
+      <section class="page__section" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="contacts__content">
+            <form
+              class="contacts__form form"
+              action="#"
+              id="#form"
+            >
+              <input
+                class="form__field"
+                name="name"
+                type="text"
+                required
+                placeholder="Name"
+                pattern="[A-Za-z]{1,32}"
```

Comment:
Shallow range of possible characters. What about other languages, spaces, etc.? Better omit this pattern or find a better regexp.

Reply:
fixed